include/wlr/types/wlr_virtual_keyboard.h

```diff
@@ -0,0 +1,28 @@
+#ifndef WLR_TYPES_WLR_VIRTUAL_KEYBOARD_H
+#define WLR_TYPES_WLR_VIRTUAL_KEYBOARD_H
```

Comment:
All of the symbols should be include the unstable version too.

Reply:
What do you mean?

Reply:
This protocol is `virtual_keyboard_unstable_v1`. All symbols should be chosen so that there's no conflict when a new unstable version is published (e.g. `virtual_keyboard_unstable_v2`) or when the stable version is adopted (`virtual_keyboard`).

Reply:
Does the file name need to change?

Reply:
Yes.